pkg/web/src/hooks.client.ts

```diff
@@ -0,0 +1,40 @@
+import {getWebInstrumentations, initializeFaro} from '@grafana/faro-web-sdk';
+import {TracingInstrumentation} from '@grafana/faro-web-tracing';
+import {PUBLIC_BACKEND_ENDPOINT} from '$env/static/public'
+
+function setupFaro() {
+    const res = fetch(`${PUBLIC_BACKEND_ENDPOINT}api/config`).
```

Comment:
I don't think res is being used, so we can remove it.

```suggestion
    fetch(`${PUBLIC_BACKEND_ENDPOINT}api/config`).
```

Reply:
Good catch there!